CSS/style.css

```diff
@@ -426,6 +461,111 @@ body {
   margin: 18px 0;
 }
 
+.modal {
+  background: #fff;
+  padding: 5.5rem;
+  width: 90%;
+  opacity: 0;
+  visibility: hidden;
+  transition: transform 1s, opacity 1s, visibility 0s;
+  z-index: 300;
+  border-radius: 1.5rem;
+}
```

Comment:
- [ ] On some mobile screen sizes your modal is larger than the screen and it is possible to scroll down, which is not a good user experience. It also has huge padding on its sides making the content inside not well placed and breaking like the buttons on the bottom. Please fix the responsiveness of the modal to fit perfectly on every mobile screen size and look as much as possible as the Figma template.
 - [ ] On the desktop view of larger screens (width > 1400px) something similar happens with the buttons. It also has a lot of whitespace around the modal. Kindly try to fix the responsiveness of your modal so it can fit properly on any kind of screen.
 -Iphone SE view:
<img width="490" alt="image" src="https://user-images.githubusercontent.com/61919617/188915360-abb511e4-3dca-4333-bedd-1326f02cfeae.png">
-Desktop view::
<img width="1451" alt="image" src="https://user-images.githubusercontent.com/61919617/188916018-3fb8c547-207d-4aab-be8e-731ef6282094.png">